pp/integrations/plugin-packs/procedures/operatingsystems-linux-snmpv3.md

```diff
@@ -0,0 +1,965 @@
+| --critical-*           | Critical threshold. Can be: 'read', 'write', 'read-iops', 'write-iops', 'total-read', 'total-write', 'total-read-iops', 'total-write-iops', 'sum-read-write', 'sum-read-write-iops'.                                                          |
+| --device               | Set the device (number expected) example: 1, 2,... (empty means 'check all devices').                                                                                                                                                         |
+| --name                 | Allows to use device name with option --device instead of devoceoid index.                                                                                                                                                                    |
+| --regexp               | Allows to use regexp to filter devices (with option --name).                                                                                                                                                                                  |
+| --regexp-insensitive   | Allows to use regexp non case-sensitive (with --regexp).                                                                                                                                                                                      |
+
+</TabItem>
+<TabItem value="Inodes-Global" label="Inodes-Global">
+
+| Option                                          | Description                                                                                                                                                                                        |
+|:------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --warning-usage                                 | Warning threshold in percent.                                                                                                                                                                      |
+| --critical-usage                                | Critical threshold in percent.                                                                                                                                                                     |
+| --diskpath                                      | Set the disk path (number expected) example: 1, 2,... (empty means 'check all disks path').                                                                                                        |
+| --name                                          | Allows to use disk path name with option --diskpath instead ofdisk path oid index.                                                                                                                 |
```

Comment:
```suggestion
| --name                                          | Allows to use disk path name with option --diskpath instead of disk path OID index.                                                                                                                 |
```